Doc/reference/datamodel.rst

```diff
@@ -2043,10 +2044,10 @@ Basic customization
 
 .. method:: object.__str__(self)
 
-   Called by :func:`str(object) <str>` and the built-in functions
-   :func:`format` and :func:`print` to compute the "informal" or nicely
+   Called by :func:`str(object) <str>`, the default :meth:`__format__` implementation,
```

Comment:
Does this role (without class reference) create a proper link to the \_\_format\_\_ method doc?

Reply:
When you click on  `:meth:`__format__`, it directs you to `object.__format__(self, format_spec)`